src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Select.js

```diff
@@ -43,10 +44,45 @@ class Select<T> extends React.Component<Props<T>> {
 
     @observable selectedOptionRef: ?ElementRef<'li'>;
 
+    @observable buttonRefsByIndex: Map<number, ElementRef<'button'>> = new Map();
+
+    @observable searchText: string = '';
+
+    @observable focusedElementIndex: number = -1;
+
     @observable open: boolean;
 
+    @computed get buttonTexts(): Map<number, string> {
```

Comment:
now we need to call this `buttonTextsByIndex` 😇

Reply:
fixed